SecurityPkg/Include/Guid/Tpm2ServiceFfa.h

```diff
@@ -0,0 +1,43 @@
+/** @file
+  Provides function interfaces to communicate with TPM 2.0 service through FF-A.
+
+  This header follows the TPM over FF-A specification:
+  https://developer.arm.com/documentation/den0138/latest/
+
+  Copyright (c), Microsoft Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef TPM2_SERVICE_FFA_H_
+#define TPM2_SERVICE_FFA_H_
+
+#define EFI_TPM2_SERVICE_FFA_GUID \

```

Comment:
This GUID is defined in TPM over FF-A spec section 3.1

Then you don't need to add `EFI_` prefix. I recommend to use `TPM2_FFA_SERVICE_UUID`.

Reply:
Thanks, will update.